service-api/src/main/java/greencity/service/notification/ManagementNotificationService.java

```diff
@@ -0,0 +1,22 @@
+package greencity.service.notification;
+
+import greencity.dto.NotificationTemplateDto;
+import greencity.dto.PageableDto;
+import org.springframework.data.domain.Pageable;
+
+public interface ManagementNotificationService {
+    /**
```

Comment:
what is the purpose of specifying @ inheritDoc everywhere? these methods are custom repo methods. They won't inherit any documentation. right?

Reply:
mvn checkstyle:check gives an error if there is no documentation

Reply:
please add meaningful javadoc instead.